.github/workflows/tests.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [3.6, 3.7, 3.8, 3.9]
+        python: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
The really horrible annoying fact is that github actions doesn't like `3.10`: it needs to be in quotes.

Reply:
> The really horrible annoying fact is that github actions doesn't like `3.10`: it needs to be in quotes.

I guess all the version should be in quotes that would be better

Reply:
That sounds good for consistency. My guess is that they're being interpreted as floating point numbers, which means stripping trailing zeros.

Reply:
Yeah , it would be great if all the python version should be in quotes